devel/api.md

```diff
@@ -625,6 +627,37 @@ Type: `integer`
 
 Ethernet speed in the unit of Mbps.
 
+## `Ethernet.SRIOV_SUBTREE`
+
+Besides basic ethernet interface properties, if the ethernet device support
```

Comment:
Suggestion:

```
If the ethernet interface supports single root input/output virtualization (SR-IOV), the `Ethernet.SRIOV_SUBTREE` dictionary allows to configure it. Nmstate supports the following options:
```

- it can be a little bit more shorter
- it is not relevant whether it is configured because a user might configure it for the first time with Nmstate
- add the SR-IOV abbreviation

Reply:
Yes that is better. I will also add the SR-IOV abbreviation. Thanks!

Reply:
Done!